packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -843,6 +848,7 @@ class AreaPaneHeader extends StatelessWidget implements PreferredSizeWidget {
             scrollDirection: Axis.horizontal,
             children: scrollableCenterActions,
           ),
+          key: const Key('scrollableCenterActionsContainer'),
```

Comment:
it we are only adding this key for testing, we should remove it and find another way to check the center actions' existence in tests

Reply:
since i want to test the lack of existence I think I need to keep a key so that I have something to test against.
From my experience in UI testing with html-ish stuff adding identifiers/keys was pretty standard practice. Do we have a different way of doing this in flutter?

Reply:
ah thank you for sharing that video about keys. I had just assumed it was something akin to css ids/classes but see that it goes much deeper now.
I will approach this in the left right centre fashion in order to clean this up and make things less sketchy for testing too :D